.github/workflows/image.yml

```diff
@@ -112,3 +111,40 @@ jobs:
           provenance: mode=max
           sbom: true
           push: true
+
+  create-manifest:
+    name: Create multi-platform manifest
+    needs: build-main-or-release
+    if: |
+      github.repository_owner == 'getsentry'
+      && github.event_name == 'push'
+      && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) || startsWith(github.ref, 'refs/heads/release/'))
```

Comment:
Do we actually even need this condition, since it already needs `build-main-or-release`